sdk/digitaltwins/Azure.DigitalTwins.Core/tests/DigitalTwinsTestEnvironment.cs

```diff
@@ -8,10 +8,10 @@ namespace Azure.DigitalTwins.Core.Tests
     public class DigitalTwinsTestEnvironment : TestEnvironment
     {
         public DigitalTwinsTestEnvironment()
-            : base("digitaltwins")
+            : base(TestSettings.AdtEnvironmentVariablesPrefix.ToLower())
```

Comment:
@azabbasi - Checking to see if this makes sense. This code path has changed so want to ensure this still holds good. I want to re-purpose the constant we have so that if we change that, it updates everything else.